From: =?utf-8?q?Jonas_=C3=85dahl?= <jadahl@gmail.com>
Date: Fri, 14 Sep 2018 18:50:29 +0200
Subject: region-utils: Some whitespace fixes

Related: https://gitlab.gnome.org/GNOME/mutter/issues/300
(cherry picked from commit 0ae7ef4bc4769f14bf957c3787cc072f20d71bea)
---
 src/compositor/region-utils.h | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/src/compositor/region-utils.h b/src/compositor/region-utils.h
index 526e6ec..c9270c3 100644
--- a/src/compositor/region-utils.h
+++ b/src/compositor/region-utils.h
@@ -92,11 +92,12 @@ void     meta_region_iterator_init      (MetaRegionIterator *iter,
 gboolean meta_region_iterator_at_end    (MetaRegionIterator *iter);
 void     meta_region_iterator_next      (MetaRegionIterator *iter);
 
-cairo_region_t *meta_region_scale (cairo_region_t *region, int scale);
+cairo_region_t * meta_region_scale (cairo_region_t *region,
+                                    int             scale);
 
-cairo_region_t *meta_make_border_region (cairo_region_t *region,
-                                         int             x_amount,
-                                         int             y_amount,
-                                         gboolean        flip);
+cairo_region_t * meta_make_border_region (cairo_region_t *region,
+                                          int             x_amount,
+                                          int             y_amount,
+                                          gboolean        flip);
 
 #endif /* __META_REGION_UTILS_H__ */
